src/utils/checks/oneTxMustBeUpgraded.ts

```diff
@@ -0,0 +1,13 @@
+import { Extension, ONE_TX_PAYMENT_VERSION_LATEST } from '@colony/colony-js';
```

Comment:
@rdig I didn't know this constant exists, how does it relate to storing the most recent version available in the network in the DB and exposing them on extension data via `availableVersion` field?
In the dapp it was yet something different, we were checking for a specific OneTxPayment version...

Reply:
Good point @willm30. Raul wanted to keep the versions in the database. I guess we'll find out once he's back

Reply:
@rdig any thoughts on the comments above?

Reply:
@jakubcolony @willm30 discussed with Raul and the extension version should come from the DB but for now we can leave it as is until #163 is merged. I left a comment in the code.

Reply:
@Julianahlk #163 is keeping track of the Colony version, you should be able to get the extension version from the db in this branch

Reply:
Oh okay cool. I will look into it. Thanks 👍

Reply:
@jakubcolony and @willm30 I used `GetCurrentExtensionVersion` query as suggested. Can you both please take a look again? Thank you.